### DISCUSSION TITLE
How to allow YOLO to take duplicate captures

### DISCUSSION THREAD

**Discussion by mermetal** [ultralytics/ultralytics#9488] [Q&A]
![image](https://github.com/ultralytics/ultralytics/assets/110658137/3761c8e7-edbe-4579-9831-4598bd6a344a)
hello

As shown in the photo above, I would like to be able to detect overlapping bounding boxes with yolov5 or yolov8.

The red box is person, and the yellow box is violence.

Is there a way to allow yolo to detect duplicates in this way?

**Comment by pderrenger** [comment]
Hello! 😊

Certainly, YOLO models, including YOLOv5 and YOLOv8, are capable of detecting overlapping bounding boxes as shown in your example with the person (red box) and an action like violence (yellow box).

To achieve this, ensure your dataset properly labels the objects and any actions or additional categories in separate annotations for the same image. Here's a simplified example of what your label file could look like for one image:

```
# For an image with a person and a violent action overlapping
0 x_center y_center width height  # Person class
1 x_center y_center width height  # Violence class
```
Make sure `x_center`, `y_center`, `width`, and `height` are normalized to the image size and represent the bounding boxes for the person and the violence occurring in the image. The process would be the same for YOLOv8. When you train your model on a dataset annotated in this way, it will learn to detect both objects and actions even if they overlap.

Just ensure your classes (e.g., person, violence) are well represented in your dataset for the best detection performance. 🚀
